Jenkinsfile-build

```diff
@@ -103,19 +102,19 @@ pipeline {
     }
   }
   post {
-      failure {
-        node('master') {
-          script {
-            tdr.reportFailedBuildToGitHub(repo, env.GIT_COMMIT)
-          }
+    failure {
+      node('master') {
+        script {
+          tdr.reportFailedBuildToGitHub(repo, env.GIT_COMMIT)
         }
       }
-      success {
-        node('master') {
-          script {
-            tdr.reportSuccessfulBuildToGitHub(repo, env.GIT_COMMIT)
-          }
+    }
+    success {
+      node('master') {
```

Comment:
Just noticed this because I fixed it in tdr-frontend yesterday: you can drop the `node` specification in the `post` steps when it's the same as the default one, and it helps avoid the Jenkins deadlocks we sometimes get.

Reply:
Fixed